test/controllers/GetInterestsControllerSpec.scala

```diff
@@ -32,7 +32,7 @@ class GetInterestsControllerSpec extends TestSuite{
   val mtditid: String = "1234567890"
   val taxYear: String = "2020"
   val service: GetInterestsService = mock[GetInterestsService]
-  val controller = new GetInterestsController(mockControllerComponents, authorisedAction, service)
+  val controller = new GetInterestsController(mockControllerComponents, authorisedAction, service)(mockExecutionContext)
```

Comment:
why did you add this in?

Reply:
Thought I had an error that was saying it needed the implicit declared but it was something else in the end.